news/2019-09-20-new-featured-artist-koan-sound.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "New Featured Artist: KOAN Sound"
+date: 2019-09-20 01:00:00 +0000
+---
+
+We're proud to welcome **KOAN Sound** aboard as our latest Featured Artist! 16 tracks now available, including all of their latest album, *Intervals Above*!
+
+![](https://assets.ppy.sh/artists/49/header.jpg)
+
+Showcased in osu! since early 2015 by [Aiceo](https://osu.ppy.sh/users/4453280)'s set of [Starlite](https://osu.ppy.sh/beatmapsets/294496#osu/662054) (featuring Asa), **KOAN Sound** has been something of an underground hit in the osu! community among the mapping scenes and abroad. Further from home, the enigmatic duo has toured extensively worldwide, supporting headline acts like *Skrillex*, *Pretty Lights* and more. They're also currently on tour at the moment around the United States with *Haywyre* in a show titled 'In Parallel'.
```

Comment:
```suggestion
Showcased in osu! since early 2015 by [Aiceo](https://osu.ppy.sh/users/4453280)'s set of [Starlite](https://osu.ppy.sh/beatmapsets/294496) (featuring Asa), **KOAN Sound** has been something of an underground hit in the osu! community among the mapping scenes and abroad. Further from home, the enigmatic duo has toured extensively worldwide, supporting headline acts like *Skrillex*, *Pretty Lights* and more. They're also currently on tour at the moment around the United States with *Haywyre* in a show titled "In Parallel".
```